msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,626 @@
+
+    The following JSON is an example:
+
+    ```
+    "short": "Contoso Tab and Add-in-${{TEAMSFX_ENV}}",
+    ```
+
+    > [!NOTE]
+    > The "name.short" value appears in both the Teams tab capability and the Outlook add-in. Examples: 
+    >
+    > - It is the label under the launch button of the Teams tab.
+    > - It is content of the title bar of the add-in's task pane.
+
+1. If you changed the "name.short" value from its default (which is the name of the project followed by the `${{TEAMSFX_ENV}}` variable), make exactly the same change in all places where the project name appears in the following two files in the root of the project: teamsapp.yml and teamsapp.local.yml.
+1. If there is no "authorization.permissions.resourceSpecific" array in the Teams manifest template, copy it from the add-in manifest as a top-level property. If there already is one in the Teams template, copy any objects from the array in the add-in manifest to the array in the Teams template. The following JSON is an example:
+
+    ```json
+    "authorization": {
+        "permissions": {
+            "resourceSpecific": [
+                {
+                    "name": "MailboxItem.Read.User",
+                    "type": "Delegated"
+                }
+            ]
+        }
+    },
+    ```
+
+1. In the env/.env.local file, find the lines that assign values to the `TAB_DOMAIN` and `TAB_ENDPOINT` variables. Add the following lines immediately below them:
+
+    ```
+    ADDIN_DOMAIN=localhost:3000
+    ADDIN_ENDPOINT=https://localhost:3000
+    ```
+
+1. In the env/.env.dev file, add the following line just below the `TAB_ENDPOINT=` ... line:
+
+    ```
+    ADDIN_ENDPOINT=
+    ```
+
+1. In the Teams manifest template, add the placeholder `"${{ADDIN_DOMAIN}}",` to the top of the "validDomains" array. The Teams Toolkit replaces this with a localhost domain when you're developing locally. When you deploy the finished combined app to staging or production as described in [Move the application to Azure](#move-the-application-to-azure), Teams Toolkit replaces the placeholder with the staging/production URI. The following JSON is an example:
+
+    ```json
+    "validDomains": [
+        "${{ADDIN_DOMAIN}}",
+        
+        // other domains or placeholders
+    ],
+    ```
+
+1. Copy the entire "extensions" property from the add-in's manifest into the Teams app manifest template as a top-level property.
```

Comment:
```suggestion
1. Copy the entire `extensions` property from the add-in's manifest into the Teams app manifest template as a top-level property.
```

Reply:
This change is not consistent with the standard practice in Teams docs, which is to put manifest property names in quotation marks.

Reply:
Agreed. Incorrect comment, please ignore.